phpstan.neon.dist

```diff
@@ -5,11 +5,6 @@ parameters:
         - src
     bootstrapFiles:
         - tests/bootstrap.php
-    ignoreErrors:
-        -
-            count: 1
-            message: '~^Undefined variable: \$phpCodeSnifferConfig$~'
-            path: src/Config.php
```

Comment:
*grin* ... well, this _was_ needed until three hours ago.... Looks like PHPStan 1.10.48 fixed this. It still threw that false positive in 1.10.47.

https://github.com/phpstan/phpstan/releases/tag/1.10.48

Reply:
Would you be open to a change to the `require-dev` section of `composer.json` to set a minimum version for `phpstan` as part of this pull request?

Reply:
PHPStan is not included in the `composer.json` file and should not be either as it - and its dependencies - can interfere with PHPCS itself, causing test breakage or test passes, while things shouldn't pass.

For this project, PHPStan should only ever be used via a PHAR file and that's what's being done in the workflow (via the `setup-php` action runner).